docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3.8"
+
+services:
+  backend:
+    build:
+      context: .
+      dockerfile: backend/Dockerfile
```

Comment:
could that be `context: backend/Dockerfile`

Reply:
I think it should be root
Otherwise the misc folder won't persist from the outside, it sill reset all newly uploaded files, right?